cromwellApiClient/src/main/scala/cromwell/api/model/CromwellQueryResult.scala

```diff
@@ -7,9 +7,9 @@ import cromwell.api.model.WorkflowStatusJsonFormatter._
 
 case class CromwellQueryResults(results: Seq[CromwellQueryResult])
 
-case class CromwellQueryResult(name: String, id: WorkflowId, status: WorkflowStatus, end: OffsetDateTime, start: OffsetDateTime)
+case class CromwellQueryResult(name: String, id: WorkflowId, status: WorkflowStatus, end: OffsetDateTime, start: OffsetDateTime, metadataArchiveStatus: String)
```

Comment:
The use of this case class for checking archive status appears to be responsible for most (definitely not all) of the remaining failures on this branch. For submissions that did not parse the `name` field will be `null` which leads to a missing key/value pair in the response JSON. `name` is a `String` and not an `Option[String]` in the case class and the current deserializer setup does not deal with this gracefully.

```
- should fail during execution import_passwd *** FAILED *** (1 minute, 47 seconds)
  spray.json.DeserializationException: Object is missing required member 'name'
  at spray.json.package$.deserializationError(package.scala:23)
  at spray.json.ProductFormats.fromField(ProductFormats.scala:63)
  at spray.json.ProductFormats.fromField$(ProductFormats.scala:51)
  at cromwell.api.model.CromwellQueryResultJsonSupport$.fromField(CromwellQueryResult.scala:12)
  at spray.json.ProductFormatsInstances$$anon$1.read(ProductFormatsInstances.scala:37)
  at spray.json.ProductFormatsInstances$$anon$1.read(ProductFormatsInstances.scala:29)
  at akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport.$anonfun$sprayJsonUnmarshaller$1(SprayJsonSupport.scala:30)
  at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
  at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
  at akka.http.scaladsl.util.FastFuture$.transformWith$extension1(FastFuture.scala:45)
  ...
  Cause: java.util.NoSuchElementException: key not found: name
  at scala.collection.MapLike.default(MapLike.scala:235)
  at scala.collection.MapLike.default$(MapLike.scala:234)
  at scala.collection.immutable.TreeMap.default(TreeMap.scala:50)
  at scala.collection.MapLike.apply(MapLike.scala:144)
  at scala.collection.MapLike.apply$(MapLike.scala:143)
  at scala.collection.immutable.TreeMap.apply(TreeMap.scala:50)
  at spray.json.ProductFormats.fromField(ProductFormats.scala:58)
  at spray.json.ProductFormats.fromField$(ProductFormats.scala:51)
  at cromwell.api.model.CromwellQueryResultJsonSupport$.fromField(CromwellQueryResult.scala:12)
  at spray.json.ProductFormatsInstances$$anon$6.read(ProductFormatsInstances.scala:162)

```